src/gwc/-u

```diff
@@ -0,0 +1,27 @@
+Note: Unnecessary use of -X or --request, GET is already inferred.
```

Comment:
I don't think this was meant to be included in the PR :rofl:

Reply:
fixed